test/livebook/session_test.exs

```diff
@@ -2092,6 +2093,104 @@ defmodule Livebook.SessionTest do
     end
   end
 
+  describe "code evaluation logging" do
```

Comment:
@jonatanklosko, this is the first time I'm writing tests related to creating notebooks, cells, and apps. Please check if the code is okay.